app/src/main/java/com/phivle/fitnesstasks/fragments/SummaryFragment.java

```diff
@@ -0,0 +1,97 @@
+package com.phivle.fitnesstasks.fragments;
+
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.phivle.fitnesstasks.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * {@link SummaryFragment.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link SummaryFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class SummaryFragment extends Fragment {
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
+    private static final String ARG_PARAM2 = "param2";
+
+    // TODO: Rename and change types of parameters
+    private String mParam1;
+    private String mParam2;
+
+    private OnFragmentInteractionListener mListener;
+
+    public SummaryFragment() {
+        // Required empty public constructor
+    }
+
+    /**
+     * Use this factory method to create a new instance of
+     * this fragment using the provided parameters.
+     *
+     * @param param1 Parameter 1.
+     * @param param2 Parameter 2.
+     * @return A new instance of fragment SummaryFragment.
+     */
+    // TODO: Rename and change types and number of parameters
+    public static SummaryFragment newInstance(String param1, String param2) {
```

Comment:
I'm almost in favor of leaving this function call without parameters, and setting the fragment arguments with an otherwise empty Bundle `args`. That way we won't have to commit constants we don't really need (`ARG_PARAM1`, `ARG_PARAM2`) _and_ we can cut down on the number of todos we commit to the codebase.